src/back/server.py

```diff
@@ -6,171 +6,133 @@
-                cv2.THRESH_BINARY,
-                9,
-                2,
-            )
-            img_edges = cv2.cvtColor(img_edges, cv2.COLOR_GRAY2RGB)
-
-            # combine color and edges
-            img = cv2.bitwise_and(img_color, img_edges)
-
-            # rebuild a VideoFrame, preserving timing information
-            new_frame = VideoFrame.from_ndarray(img, format="bgr24")
-            new_frame.pts = frame.pts
-            new_frame.time_base = frame.time_base
-            return new_frame
-        elif self.transform == "edges":
-            # perform edge detection
-            img = frame.to_ndarray(format="bgr24")
-            img = cv2.cvtColor(cv2.Canny(img, 100, 200), cv2.COLOR_GRAY2BGR)
-
-            # rebuild a VideoFrame, preserving timing information
-            new_frame = VideoFrame.from_ndarray(img, format="bgr24")
-            new_frame.pts = frame.pts
-            new_frame.time_base = frame.time_base
-            return new_frame
-        elif self.transform == "rotate":
-            # rotate image
-            img = frame.to_ndarray(format="bgr24")
-            rows, cols, _ = img.shape
-            M = cv2.getRotationMatrix2D((cols / 2, rows / 2), frame.time * 45, 1)
-            img = cv2.warpAffine(img, M, (cols, rows))
-
-            # rebuild a VideoFrame, preserving timing information
-            new_frame = VideoFrame.from_ndarray(img, format="bgr24")
-            new_frame.pts = frame.pts
-            new_frame.time_base = frame.time_base
-            return new_frame
-        elif self.transform == "skeleton":
-            # Convert the aiortc frame to an array
-            img = frame.to_ndarray(format="bgr24")
-
-            # Process the frame for skeleton
-            img = await process_frame_for_skeleton(img)
-
-            # Rebuild a VideoFrame, preserving timing information
-            new_frame = VideoFrame.from_ndarray(img, format="bgr24")
-            new_frame.pts = frame.pts
-            new_frame.time_base = frame.time_base
-            return new_frame
-        else:
-            return frame
+source_video = VideoStreamTrack()
+NO_TRANSFORM: str = "none"
+__transformed_tracks = dict()
+
+
+def apply_transform(track: MediaStreamTrack, transform: str) -> MediaStreamTrack:
+    if transform == NO_TRANSFORM:
+        return track
+    if transform not in __transformed_tracks:
+        __transformed_tracks[transform] = VideoTransformTrack(track, transform)
+    return __transformed_tracks[transform]
+
+
+consumer_schema = {
+    "type": "object",
+    "properties": {
+        "sdp": {"type": "string"},
+        "type": {"type": "string"},
+        "video_transform": {"type": "string"},
+    },
+    "required": ["sdp", "type", "video_transform"],
+}
+
+broadcast_schema = {
+    "type": "object",
+    "properties": {
+        "sdp": {"type": "string"},
+        "type": {"type": "string"},
+    },
+    "required": ["sdp", "type"],
+}
+
 
 async def consumer(request):
```

Comment:
Please name the broadcast and consume function in a consistent manner, either "broadcast" vs "consume" or "broadcaster" vs "consumer"